src/utils.js

```diff
@@ -88,7 +59,7 @@ function trimSiaPrefix(str) {
 module.exports = {
   defaultOptions,
   defaultPortalUrl,
-  executeRequest,
+  defaultSkynetPortalUrl,
```

Comment:
Do we need this, if we already have `defaultPortalUrl`?

Reply:
`defaultPortalUrl` will be extended in the future to be smarter picking siasky.net only as a last result. I think it will still be useful to provide the de facto Skynet portal.

Reply:
But it will still always provide a valid portal URL, right, even if it returns siasky.net only as last resort, right?

I don't mind leaving it in but if it's available, and especially if it returns something different than the result of `defaultPortalUrl` then we should add some docstrings explaining the exact difference and when to use which one.